docs/topics/db/managers.txt

```diff
@@ -63,39 +63,28 @@ returns a list of all ``OpinionPoll`` objects, each with an extra
 ``num_responses`` attribute that is the result of an aggregate query::
 
     from django.db import models
+    from django.db.models.functiosn import Coalesce
```

Comment:
Sorry just dropping in here - happened to notice typo here `functiosn` -> `functions`.

Reply:
Thanks, fixed.